docs/source/using/static-ip.adoc

```diff
@@ -0,0 +1,72 @@
+include::variables.adoc[]
+
+= Assign Static IP Address
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[static-ip-overview]]
+== Overview
+
+include::con_static-ip-overview.adoc[]
+
+[NOTE]
+====
+- Assigning a static IP address to the {project} VM is only officially supported for Hyper-V.
+- The {project} VM cannot be assigned a static IP address when using the KVM driver plug-in.
+- An experimental `--set-static` flag is available for the `minishift ip` command for other hypervisors.
+See xref:../using/experimental-features.adoc#set-fixed-ip[Set Fixed IP Address] for more information.
+====
+
+[[static-ip-hyperv]]
+== Assign IP Address to Hyper-V
+
+Since the Internal Virtual Switch for Hyper-V does not provide a DHCP offer option, an IP address needs to be provided in a different way.
+Functionality is provided to assign an IP address on startup using the Data Exchange Service for Hyper-V.
+
+[IMPORTANT]
+====
+This only works with the CentOS or RHEL xref:../using/basic-usage.adoc#choosing-iso-image[ISO] in combation with Hyper-V.
+The Boot2Docker ISO image experiences a problem when the values are being sent to the {project} instance and consumed by the Boot2Docker ISO.
+We are looking into the issue and hope to provide a solution in the future.
+====
+
+to make this work, you need to create a link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/setup-nat-network[Virtual Switch using NAT].
+
+[NOTE]
+====
+WinNAT is limited to one NAT network per host.
+For more details about capabilities and limitations, please see the link:https://blogs.technet.microsoft.com/virtualization/2016/05/25/windows-nat-winnat-capabilities-and-limitations/[WinNAT capabilities and limitations blog].
+====
+
+The following command will attempt to assign an IP address for use on the Internal Virtual Switch 'MyInternal':
+
+----
+PS> $env:HYPERV_VIRTUAL_SWITCH="MyInternal"
+PS> minishift.exe start `
+  --iso-url centos `
```

Comment:
I know this is migrated, but we might not want to mention `centos` here, as this will also end up in the downstream documentation without modifications. WDYT? In a note we have already mentioned it only works on these ISOs

Reply:
Good point. I briefly thought about this as well while migrating it, but also considered whether the command being run would work upstream without the use of `--iso-url`. Either way is fine with me, whether we make changes here to remove that reference or do that downstream.

Reply:
downstream actually supports the `--iso-url` flag, but it is not necessary as the ISO is included and set as default.

Reply:
@gbraad is right. That means in downstream we should not use the `--iso-url ` flag at all (i.e. examples)